src/backend/kubernetes_backend.py

```diff
@@ -37,9 +36,8 @@
 logger = logging.getLogger("KubernetesBackend")
 
 
-class KubernetesBackend(BackendInterface):
+class KubernetesBackend:
     def __init__(self, config_dir: Path, network_name: str, logs_pod="fluentd") -> None:
```

Comment:
I think we can get rid of `config_dir: Path`

Reply:
I think I'll leave this for now in this changeset, but will add it to my list of followups [above](https://github.com/bitcoin-dev-project/warnet/pull/382#issuecomment-2166684837)